website/docs/d/lb_service_monitor.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_lb_service_monitor"
```

Comment:
maybe add `data`

Reply:
I think it is not needed as the `data` is more of terraform keyword like `resource` and website menu clearly separates these objects. Open to discussion.

Reply:
cc: @lvirbalas

Reply:
Where to add `data` @vbauzysvmware ?

Reply:
I am not sure: `"vCloudDirector: data source vcd_lb_service_monitor"`, "vCloudDirector: data vcd_lb_service_monitor", "vCloudDirector: data_vcd_lb_service_monitor"

Reply:
But is it for like SEO reasons? This field sets only page title (name of browser tab). Rarely people see it I guess. While browsing you see the location quite clearly (or so it looks to me)

![image](https://user-images.githubusercontent.com/15804230/59427374-29ce7680-8de3-11e9-9339-d6eaa7ec63fe.png)

Reply:
Agreed to go with naked names so far.